### DISCUSSION TITLE
Eebus interface

### DISCUSSION THREAD

**Discussion by BeeJoo123** [tbnobody/OpenDTU#2341] [Ideas]
Da immer mehr Energie Management Systeme den Eebus verwenden, wäre es schön, wenn die OpenDTU ebenfalls eine solche Schnittstelle hätte. Dann könnte man in diesen Systemen einfache Übersichten haben und z.B. die Wallbox einschalten, wenn die PV Anlage genügend Strom liefert etc. Kann man da etwas erwarten ?

**Comment by ghost** [comment]
Hi,
ich kann's nicht beantworten, interessiert mich aber auch, daher:
... push ...
